<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="SliderWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1080</width>
      <height>290</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PySlider" name="DefrostSlider">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border-radius: 1px;
    height: 3px;
    margin: 0px;
    background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
    background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 44px;
    width: 44px;
    margin: -20px 0;
    border-radius: 20px;
    padding: -20px 0px;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PySlider" name="VentSlider">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border-radius: 1px;
    height: 3px;
    margin: 0px;
    background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
    background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 44px;
    width: 44px;
    margin: -20px 0;
    border-radius: 20px;
    padding: -20px 0px;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="PySlider" name="TempSlider">
        <property name="minimumSize">
         <size>
          <width>820</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>820</width>
          <height>44</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border-radius: 1px;
    height: 3px;
    margin: 0px;
    background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
    background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 44px;
    width: 44px;
    margin: -20px 0;
    border-radius: 20px;
    padding: -20px 0px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="PyGlView" name="CarView" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>360</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>720</height>
     </size>
    </property>
   </widget>
   <widget class="PyGlView" name="CoolFanView" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>360</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>720</height>
     </size>
    </property>
   </widget>
   <widget class="PyGlView" name="HeatFanView" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>720</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PySlider</class>
   <extends>QSlider</extends>
   <header>py_widget.h</header>
  </customwidget>
  <customwidget>
   <class>PyGlView</class>
   <extends>QWidget</extends>
   <header>py_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
